tiflash/tiflash-supported-pushdown-calculations.md

```diff
@@ -150,3 +155,52 @@ SHOW WARNINGS;
+|             └─Projection_10      | 10000.00 | mpp[tiflash] |               | test.t.id, gt(test.t.id, 10)->Column#4                                                                        |
+|               └─TableFullScan_11 | 10000.00 | mpp[tiflash] | table:t       | keep order:false, stats:pseudo                                                                                |
++----------------------------------+----------+--------------+---------------+---------------------------------------------------------------------------------------------------------------+
+9 rows in set (0.0073 sec)
+
+```
+
+In this output you can see that the `Window` operation has a value of `mpp[tiflash]` in the `task` column indicating it was able to push down the `ROW_NUMBER() OVER (PARTITION BY id > 10)` operation to TiFlash.
+
+```sql
+CREATE TABLE t(id INT PRIMARY KEY, c1 VARCHAR(100));
+ALTER TABLE t SET TIFLASH REPLICA 1;
+INSERT INTO t VALUES(1,"foo"),(2,"bar"),(3,"bar foo"),(10,"foo"),(20,"bar"),(30,"bar foo");
+
+EXPLAIN SELECT id, MAX(id) OVER (PARTITION BY id > 10) FROM t;
++-----------------------------+----------+-----------+---------------+------------------------------------------------------------+
+| id                          | estRows  | task      | access object | operator info                                              |
++-----------------------------+----------+-----------+---------------+------------------------------------------------------------+
+| Projection_6                | 10000.00 | root      |               | test.t1.id, Column#5                                       |
+| └─Shuffle_14                | 10000.00 | root      |               | execution info: concurrency:5, data sources:[Projection_8] |
+|   └─Window_7                | 10000.00 | root      |               | max(test.t1.id)->Column#5 over(partition by Column#4)      |
+|     └─Sort_13               | 10000.00 | root      |               | Column#4                                                   |
+|       └─Projection_8        | 10000.00 | root      |               | test.t1.id, gt(test.t1.id, 10)->Column#4                   |
+|         └─TableReader_10    | 10000.00 | root      |               | data:TableFullScan_9                                       |
+|           └─TableFullScan_9 | 10000.00 | cop[tikv] | table:t1      | keep order:false, stats:pseudo                             |
++-----------------------------+----------+-----------+---------------+------------------------------------------------------------+
+7 rows in set (0.0010 sec)
+```
+
+In this output you can see that the `Window` operation has a value of `root` in the `task` column indicating it wasn't able to push down the `MAX(id) OVER (PARTITION BY id > 10)` operation to TiFlash. This is because `MAX()` is only supported for push-down as aggregate function and not as window function.
```

Comment:
```suggestion
In this output, you can see that the `Window` operation has a value of `root` in the `task` column indicating it wasn't able to push down the `MAX(id) OVER (PARTITION BY id > 10)` operation to TiFlash. This is because `MAX()` is only supported for push-down as an aggregate function and not as a window function.
```